src/scribe_data/cli.py

```diff
@@ -19,3 +19,178 @@
+            max_key_length = max(len(key) for key in data.keys())
+            for key, value in data.items():
+                if isinstance(value, dict):
+                    print(f"{key:<{max_key_length}} : ")
+                    max_sub_key_length = max(len(sub_key) for sub_key in value.keys())
+                    for sub_key, sub_value in value.items():
+                        print(f"  {sub_key:<{max_sub_key_length}} : {sub_value}")
+                elif isinstance(value, list):
+                    print(f"{key:<{max_key_length}} : ")
+                    for item in value:
+                        if isinstance(item, dict):
+                            for sub_key, sub_value in item.items():
+                                print(f"  {sub_key:<{max_key_length}} : {sub_value}")
+                        else:
+                            print(f"  {item}")
+                else:
+                    print(f"{key:<{max_key_length}} : {value}")
+        elif isinstance(data, list):
+            for item in data:
+                if isinstance(item, dict):
+                    for key, value in item.items():
+                        print(f"{key} : {value}")
+                else:
+                    print(item)
+        else:
+            print(data)
+
+def query_data(language: str, word_type: str) -> None:
+    # Normalize the input language
+    normalized_language = LANGUAGE_MAP.get(language.lower())
+    if not normalized_language:
+        print(f"Language '{language}' is not recognized.")
+        return
+
+    data_file = DATA_DIR / normalized_language / f"{word_type}.json"
+    if not data_file.exists():
+        print(f"No data found for language '{normalized_language}' and word type '{word_type}'.")
+        return
+
+    try:
+        with data_file.open('r') as file:
+            data = json.load(file)
+    except (IOError, json.JSONDecodeError) as e:
+        print(f"Error reading '{data_file}': {e}")
+        return
+
+    print(f"Data for language '{normalized_language}' and word type '{word_type}':")
+    print_formatted_data(data, word_type)
+
+    if word_type.lower() == 'nouns':
+        print("\nLegend:")
+        print("PL    : Plural")
+        print("empty : Singular\n")
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description='Scribe-Data CLI Tool')
+    subparsers = parser.add_subparsers(dest='command', required=True)
+
+    subparsers.add_parser('list-languages', aliases=['ll'], help='List available languages')
+
+    list_word_types_parser = subparsers.add_parser('list-word-types', aliases=['lwt'], help='List available word types')
+    list_word_types_parser.add_argument('-l', '--language', help='Language code')
+
+    query_parser = subparsers.add_parser('query', help='Query data for a specific language and word type')
+    query_parser.add_argument('-l', '--language', required=True, help='Language code')
+    query_parser.add_argument('-wt', '--word-type', required=True, help='Word type')
```

Comment:
I'd say that for some cases we might not want to include `--language`, but that's also because I've been reconsidering the structure for #148. We have `--language` and `--word-type` as options, so maybe we could even do the following with a `list` command that uses them as arguments:

```bash
scribe-data list --language  # list all languages
scribe-data list --word-type  # list all word types
scribe-data list --language German --word-type  # list all German word types
scribe-data list --language --word-type nouns  # list all languages that you can get nouns for
```

The functionality has generally been focussed on using `--language` and `--word-type` to subset, but maybe this could also work? I'm just kind of throwing this out :) Would this make sense @mhmohona and @wkyoshida?

Regardless, for `list-word-types` `--language` wouldn't be necessary :)

Reply:
Oh my comment above was regarding the `query` command actually hahah, but yea, I do see what you're saying with the `list*` commands :thinking: I feel a bit iffy with them too.. Let's try to hash it out in #148 to what might make more sense then! 

If it makes sense - I was thinking of trying to keep this PR contained more so around getting #136 out of the way. Meaning - I'm OK if we just leave `list-languages` and `list-word-types` as is for now, while we decide on what to do